traitcuration/traits/datasources/clinvar.py

```diff
@@ -0,0 +1,77 @@
+import urllib.request
+import csv
+import gzip
+import shutil
+import itertools
+import os
+from ..models import Trait
+
+
+def extract_clinvar_data():
+    """
+    This function downloads the latest ClinVar TSV release data and extracts it into a 
+    'variant_summary.txt' file
+    """
+    print("Downloading ClinVar data...")
+    url = 'https://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/variant_summary.txt.gz'
```

Comment:
I think it's better to define this as a constant at the top of the module. Eventually this will be a configurable parameter (see https://github.com/EBIvariation/trait-curation/issues/32).